include/AutomatableModel.h

```diff
@@ -170,6 +170,16 @@ class LMMS_EXPORT AutomatableModel : public Model, public JournallingObject
 		return castValue<T>( m_value );
 	}
 
+	inline float getTrueValue(int frameOffset = 0) const
```

Comment:
It is m_value. I will rename that

Reply:
Judging by the code it could be something like `getRawValueOrControllerValue`?

Reply:
I renamed it to `getUntypedValue`, I hope it is okay

Reply:
What would "untyped" mean in this context? `m_value` is defined as type `float` and it's also what this method returns. However, it does not simply return `m_value` but adds some extra logic with regards to the controller. IMO this should be reflected in the name.

Reply:
Is `getAutomatedRawValue` right?

`m_value` is automated if I remember correctly

Reply:
If I interpret the code correctly it returns the raw value only if it is not automated. Otherwise it calls `AutomatableModel::controllerValue` which contains quite a lot of logic which does not simply return a raw value.

That's why I proposed `getRawValueOrControllerValue` because it is not `m_value` in all situations.

Reply:
> only if it is not automated
The raw value could be automated. `setAutomatedValue` changes `m_value`.
That is why i would like to call it `getAutomatedRawValue`, because m_value could be automated and raw because it isn't casted to a different type.